charts/kube-prometheus-stack/templates/grafana/dashboards-1.14/prometheus.yaml

```diff
@@ -573,6 +573,14 @@ data:
                         "stack": true,
                         "steppedLine": false,
                         "targets": [
+                            {
```

Comment:
https://github.com/prometheus-community/helm-charts/blob/e886d5ae0300be7fb51c0415d4ab4f2265301581/charts/kube-prometheus-stack/templates/grafana/dashboards-1.14/prometheus.yaml#L2-L4

Reply:
:question: This change is pulled by the sync script.

Reply:
Ping @monotek